firestore/query_multiple_inequality_test.go

```diff
@@ -148,15 +150,25 @@ func TestMultipleInequalitiesQuery(t *testing.T) {
 			Fields:     adminPbIndexFields,
 		},
 	}
+	var createdIndex *adminpb.Index
+	var waitErr error
```

Comment:
nit: I don't think you need to declare the `waitErr` here, since it's only used in the else block below.